okapi-core/src/main/java/org/folio/okapi/bean/TenantModuleDescriptor.java

```diff
@@ -33,6 +33,10 @@
     pending, deploy, invoke, undeploy, done
   }
 
+  // if true, it specifies that the id is specified by user and can not be changed
```

Comment:
This comment should be reworded, converted into javadoc and attached to the public methods getFixed() and setFixed(Boolean fixed).

`Returns true if the version is specified by user and the product can not be upgraded or downgraded.`